Cython/Compiler/Visitor.py

```diff
@@ -832,6 +832,10 @@ def repr_of(self, node):
                 result += "(type=%s, name=\"%s\")" % (repr(node.type), node.name)
             elif isinstance(node, Nodes.DefNode):
                 result += "(name=\"%s\")" % node.name
+            elif isinstance(node, ExprNodes.AttributeNode):
+                result += "(type=%s, attribute=\"%s\")" % (repr(node.type), node.attribute)
```

Comment:
It's interesting that the code in this method is explicitly calling `repr()` instead of just using `%r` for the formatting. But, in fact, it might actually be less error prone that way. Single characters are easier to get wrong than a whole function call.